# Logfile created on 2016-05-29 20:25:21 -0300 by logger.rb/53141
I, [2016-05-29T20:25:21.005815 #13255]  INFO -- : CPU Monitor Client Started!
I, [2016-05-29T20:25:21.005890 #13255]  INFO -- : Config: {"url"=>"localhost:8000", "id"=>"123456", "hostname"=>"leonam", "log_file"=>"log/cpu_monitor.log"}
RestClient.post "localhost:8000", 579 byte(s) length, "Accept"=>"application/json", "Accept-Encoding"=>"gzip, deflate", "Content-Length"=>"579", "Content-Type"=>"application/json"
E, [2016-05-29T20:25:21.044628 #13255] ERROR -- : Failed to open TCP connection to localhost:8000 (Connection refused - connect(2) for "localhost" port 8000) (Errno::ECONNREFUSED)
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient.rb:69:in `post'
/home/leonam/repo/crossover/cpu-monitor-client/lib/cpu_monitor_job.rb:21:in `perform'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:16:in `block in execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `loop'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:44:in `<top (required)>'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `start_load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:297:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/controller.rb:56:in `run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:144:in `block in run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/cmdline.rb:88:in `catch_exceptions'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:143:in `run'
cpu_monitor_control.rb:4:in `<main>'
RestClient.post "localhost:8000", 578 byte(s) length, "Accept"=>"application/json", "Accept-Encoding"=>"gzip, deflate", "Content-Length"=>"578", "Content-Type"=>"application/json"
E, [2016-05-29T20:25:26.087178 #13255] ERROR -- : Failed to open TCP connection to localhost:8000 (Connection refused - connect(2) for "localhost" port 8000) (Errno::ECONNREFUSED)
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient.rb:69:in `post'
/home/leonam/repo/crossover/cpu-monitor-client/lib/cpu_monitor_job.rb:21:in `perform'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:16:in `block in execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `loop'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:44:in `<top (required)>'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `start_load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:297:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/controller.rb:56:in `run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:144:in `block in run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/cmdline.rb:88:in `catch_exceptions'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:143:in `run'
cpu_monitor_control.rb:4:in `<main>'
RestClient.post "localhost:8000", 578 byte(s) length, "Accept"=>"application/json", "Accept-Encoding"=>"gzip, deflate", "Content-Length"=>"578", "Content-Type"=>"application/json"
E, [2016-05-29T20:25:31.133911 #13255] ERROR -- : Failed to open TCP connection to localhost:8000 (Connection refused - connect(2) for "localhost" port 8000) (Errno::ECONNREFUSED)
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient.rb:69:in `post'
/home/leonam/repo/crossover/cpu-monitor-client/lib/cpu_monitor_job.rb:21:in `perform'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:16:in `block in execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `loop'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:44:in `<top (required)>'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `start_load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:297:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/controller.rb:56:in `run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:144:in `block in run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/cmdline.rb:88:in `catch_exceptions'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:143:in `run'
cpu_monitor_control.rb:4:in `<main>'
RestClient.post "localhost:8000", 578 byte(s) length, "Accept"=>"application/json", "Accept-Encoding"=>"gzip, deflate", "Content-Length"=>"578", "Content-Type"=>"application/json"
E, [2016-05-29T20:25:36.179738 #13255] ERROR -- : Failed to open TCP connection to localhost:8000 (Connection refused - connect(2) for "localhost" port 8000) (Errno::ECONNREFUSED)
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient.rb:69:in `post'
/home/leonam/repo/crossover/cpu-monitor-client/lib/cpu_monitor_job.rb:21:in `perform'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:16:in `block in execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `loop'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:44:in `<top (required)>'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `start_load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:297:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/controller.rb:56:in `run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:144:in `block in run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/cmdline.rb:88:in `catch_exceptions'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:143:in `run'
cpu_monitor_control.rb:4:in `<main>'
RestClient.post "localhost:8000", 578 byte(s) length, "Accept"=>"application/json", "Accept-Encoding"=>"gzip, deflate", "Content-Length"=>"578", "Content-Type"=>"application/json"
E, [2016-05-29T20:25:41.223336 #13255] ERROR -- : Failed to open TCP connection to localhost:8000 (Connection refused - connect(2) for "localhost" port 8000) (Errno::ECONNREFUSED)
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient.rb:69:in `post'
/home/leonam/repo/crossover/cpu-monitor-client/lib/cpu_monitor_job.rb:21:in `perform'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:16:in `block in execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `loop'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:44:in `<top (required)>'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `start_load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:297:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/controller.rb:56:in `run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:144:in `block in run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/cmdline.rb:88:in `catch_exceptions'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:143:in `run'
cpu_monitor_control.rb:4:in `<main>'
RestClient.post "localhost:8000", 578 byte(s) length, "Accept"=>"application/json", "Accept-Encoding"=>"gzip, deflate", "Content-Length"=>"578", "Content-Type"=>"application/json"
E, [2016-05-29T20:25:46.267430 #13255] ERROR -- : Failed to open TCP connection to localhost:8000 (Connection refused - connect(2) for "localhost" port 8000) (Errno::ECONNREFUSED)
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient.rb:69:in `post'
/home/leonam/repo/crossover/cpu-monitor-client/lib/cpu_monitor_job.rb:21:in `perform'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:16:in `block in execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `loop'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:44:in `<top (required)>'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `start_load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:297:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/controller.rb:56:in `run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:144:in `block in run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/cmdline.rb:88:in `catch_exceptions'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:143:in `run'
cpu_monitor_control.rb:4:in `<main>'
RestClient.post "localhost:8000", 578 byte(s) length, "Accept"=>"application/json", "Accept-Encoding"=>"gzip, deflate", "Content-Length"=>"578", "Content-Type"=>"application/json"
E, [2016-05-29T20:25:51.314001 #13255] ERROR -- : Failed to open TCP connection to localhost:8000 (Connection refused - connect(2) for "localhost" port 8000) (Errno::ECONNREFUSED)
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
/home/leonam/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/rest-client-1.8.0/lib/restclient.rb:69:in `post'
/home/leonam/repo/crossover/cpu-monitor-client/lib/cpu_monitor_job.rb:21:in `perform'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:16:in `block in execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `loop'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:14:in `execute'
/home/leonam/repo/crossover/cpu-monitor-client/cpu_monitor.rb:44:in `<top (required)>'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:218:in `start_load'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/application.rb:297:in `start'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/controller.rb:56:in `run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:144:in `block in run'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons/cmdline.rb:88:in `catch_exceptions'
/home/leonam/.rvm/gems/ruby-2.3.0/gems/daemons-1.2.3/lib/daemons.rb:143:in `run'
cpu_monitor_control.rb:4:in `<main>'
